resources/js/beatmap-discussions/nominator.tsx

```diff
@@ -293,12 +301,26 @@ export class Nominator extends React.Component<Props> {
         <div className={`${bn}__warn`}>
           {trans('beatmapsets.nominate.dialog.hybrid_warning')}
         </div>
+        {this.renderNominatorWarning()}
       </>
     );
   }
 
   private renderModalContentNormal() {
-    return trans('beatmapsets.nominate.dialog.confirmation');
+    return (
+      <>
+        {trans('beatmapsets.nominate.dialog.confirmation')}
+        {this.renderNominatorWarning()}
```

Comment:
as in both version this is located at the end of the content and not wrapped in anything, why not just put this directly in the main render function?